paragon/_common.scss

```diff
@@ -56,3 +56,11 @@ body {
 .btn-brand {
     color: $white;
 }
+
+.edx-cookie-banner-wrapper {
```

Comment:
[nit] It's better to move it to the separate file like: `_cookie-banner.scss`.
It will save our code less complicated and more readable.

Reply:
Done